### PR TITLE
MO-7344 #time 30m Added wording for Enterprise accounts versus SparkPo…

### PR DESCRIPTION
**Pull request by ndurant-sp**
…st accounts for Dedicated IPs support document

### CODE REVIEW

File: articles/deliverability/dedicated-ip-pools.md [hunk 1]
```diff
@@ -60,4 +60,4 @@ Messages queued for delivery have the sending IP attached to the message. So the
 
 ## Scheduled Transmissions
 
-Scheduled transmissions have a slightly different behavior. If at the scheduled time, SparkPost sees that the selected IP Pool either doesn't exist or is empty, it will attempt fallback to your account's default pool if it exists. If not, it falls back to the shared pools.
+Scheduled transmissions have a slightly different behavior. If at the scheduled time, SparkPost sees that the selected IP Pool either doesn't exist or is empty, it will attempt fallback to your account's default pool if it exists. If not, it falls back to the shared pools for SparkPost accounts.  Enterprise accounts will have the transmission **rejected**.
```
Comment (rdawemsys) [inline articles/deliverability/dedicated-ip-pools.md]: It seems a bit ambiguous to me when the last sentence applies. I wonder if you should change it to be:

> If not, it falls back to the shared pools for SparkPost accounts; Enterprise accounts will have the transmission **rejected**.

or:

> If not, it falls back to the shared pools for SparkPost accounts. For Enterprise accounts, there are no shared pools and no fall back is possible, so the transmission will be **rejected**.

File: articles/deliverability/dedicated-ip-pools.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ You can provision new dedicated IPs directly to your Account via the UI.
 3. Once finished, click the "Confirm Purchases" button to purchase your dedicated IPs.
 4. The Billing page will then reflect your new purchase and your monthly invoice will be updated to reflect the charge for the dedicated IP(s). The dedicated IP(s) charge will be prorated by month.
 
-****NOTE: Each Account is limited to a maximum of 4 dedicated IPs at any one time***                                                                          
+****NOTE: Each SparkPost Account is limited to a maximum of 4 dedicated IPs at any one time. Enterpise Accounts have no limit on maximum dedicated IPs.***
```
Comment (ewandennis) [inline articles/deliverability/dedicated-ip-pools.md]: Typo: `s/Enterpise/Enterprise/`

Comment (ewandennis) [reply articles/deliverability/dedicated-ip-pools.md]: Could we split this into a **Note** of its own?

Comment (ewandennis) [reply articles/deliverability/dedicated-ip-pools.md]: `...have no limit on dedicated IPs` - remove the `maximum` in favour of the `limit`?

File: articles/deliverability/dedicated-ip-pools.md [hunk 3]
```diff
@@ -46,7 +48,7 @@ REST transmission example:
 }
 ```
 
-If you have one or more IP pools, and you do **not** provide an "ip_pool" value for your transmission, then messages will be routed through the IP(s) in the "Default" IP Pool. If the "Default" IP pool has no sending IPs then the traffic will be sent through the SparkPost shared IP pools.
+If you have one or more IP pools, and you do **not** provide an "ip_pool" value for your transmission, then messages will be routed through the IP(s) in the "Default" IP Pool. If the "Default" IP pool has no sending IPs then the traffic will be sent through the SparkPost shared IP pools for SparkPost accounts.  For Enterprise accounts, transmissions that do not specify an IP Pool and have no IPs in the "Default" IP pool will be **rejected**.
```
Comment (ewandennis) [inline articles/deliverability/dedicated-ip-pools.md]: The last sentence in this paragraph is an Enterprise caveat too. Could we split it out?

File: articles/deliverability/dedicated-ip-pools.md [hunk 4]
```diff
@@ -60,4 +62,4 @@ Messages queued for delivery have the sending IP attached to the message. So the
 
 ## Scheduled Transmissions
 
-Scheduled transmissions have a slightly different behavior. If at the scheduled time, SparkPost sees that the selected IP Pool either doesn't exist or is empty, it will attempt fallback to your account's default pool if it exists. If not, it falls back to the shared pools.
+Scheduled transmissions have a slightly different behavior. If at the scheduled time, SparkPost sees that the selected IP Pool either doesn't exist or is empty, it will attempt fallback to your account's default pool if it exists. If not, it falls back to the shared pools for SparkPost accounts.  For Enterprise accounts, there are no shared pools and no fall back is possible, so the transmission will be **rejected**.
```
Comment (ewandennis) [inline articles/deliverability/dedicated-ip-pools.md]: The last sentence in this paragraph is also an Enterprise caveat. Could we split it out too?

**Pull request merged**